src/helpers/contracts/dataShapeObject.ts

```diff
@@ -0,0 +1,20 @@
+import { DataKeys } from 'models/DataKeys'
+import data from 'data'
+
+type ValueOf<T> = T[keyof T]
+
+export default function dataShapeObject<V>(
```

Comment:
This helper looks similar to `ledgerContracts.ts`, maybe we can use only one of them? If it's too complicated to do/to use, let's leave as it is